src/main/java/de/qabel/desktop/ui/accounting/GsonContact.java

```diff
@@ -0,0 +1,23 @@
+package de.qabel.desktop.ui.accounting;
+
+import com.google.gson.JsonArray;
+import com.google.gson.annotations.Expose;
+
+
+public class GsonContact extends GsonEntity {
+
+
+	@Expose
+	public byte[] publicKey;
```

Comment:
It's public anyways. With a getter and a setter, it doesn't have to be public. Or with a public property, you don't need getters and setters. Either way, one is superfluous.